src/main/java/net/glowstone/util/GlowHelpMap.java

```diff
@@ -28,7 +28,6 @@
 import java.util.Map.Entry;
 import java.util.Set;
 import java.util.TreeMap;
-import java.util.TreeSet;
```

Comment:
Seems like TreeSet was removed instead of TreeMap? TreeSet is still used, while the usage of TreeMap was removed.

Reply:
Sorry, I will update it. It is my fault.